### DISCUSSION TITLE
CTkSlider or CTkProgressBar

### DISCUSSION THREAD

**Discussion by zSynctic** [TomSchimansky/CustomTkinter#1807] [Q&A] [answered]
I want a bar that increases when I press a button and if I would use the slider I want to remove the button and only see the bar how can I implement this whether its slider or progressbar. thanks

**Comment by AlbertSolomon** [comment, marked as answer]
**@zSynctic** you could use the progressbar widget,  such that when ever you click the button, you increment the value of the `set()` attribute of the progressbar.

```python
num = 0.1
progress_value = 0
progressbar = customtkinter.CTkProgressBar(app, orientation="horizontal")

def button_event():
    progressbar.set(progress_value + num)

button = customtkinter.CTkButton(app, text="CTkButton", command=button_event)
``` 

and you can hide the button by using `Button.pack_forget()` attribute (look for an attribute with forget in it ). _Note that you might need to debug this code because I just wrote it off the top of my head, but it contains information that might help you to solve your problem._

**Reply by zSynctic** [reply]
there was an issue with it but I fixed it, but is there a way I can set the max for the progress bar or at least change it

**Reply by AlbertSolomon** [reply]
**@zSynctic** Glad I was helpful 😊. The max value of the progress bar is always 1. ``progressbar.set(1)``
